README.md

```diff
@@ -79,3 +79,22 @@ Run with coverage `python -m pytest --cov=src`
 Run with coverage and missing lines `python -m pytest --cov-report term-missing --cov=src`
 
 Run with coverage, missing lines and junit output `python -m pytest --cov-report term-missing --junitxml="result.xml" --cov=src`
+
+## Yara rules checks.
+
+There is a Jenkins job run on a schedule from the Jenkinsfile-check-rules file. This carries out the following steps. 
+* Builds the base yara image.
+* Builds the rules yara image.
+* Gets the most recent version from git.
+* Downloads the lambda zip file from S3.
+* Unzips the zip file and copies the exiting compiled rule file into the current directory.
+* Downloads the test files from tdr-antivirus-test-files-mgmt in S3.
```

Comment:
Maybe reword slightly: "Downloads the tests file from the tdr-antivirus-test-files-mgmt S3 bucket"